[CI] CronEvalToolTests testEnsureDateIsShownInRootLocale sometimes fails

`CronEvalToolTests testEnsureDateIsShownInRootLocale` sometimes fails.  An example is https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+master+matrix-java-periodic/ES_BUILD_JAVA=java11,ES_RUNTIME_JAVA=java11,nodes=virtual&&linux/67/consoleText

The exception is:

```
   > Throwable #1: java.lang.AssertionError: 
   > Expected: a string containing "] in UTC, local time is"
   >      but: was "Valid!
   > Now is [Mon, 19 Nov 2018 05:20:04] in UTC
   > Here are the next 1 times this cron expression will trigger:
   > 1.	Mon, 2 Jan 2040 11:00:00
   > "
```

The `, local time is` bit is missing.

This did not reproduce locally for me:

```
./gradlew :x-pack:plugin:watcher:test \
  -Dtests.seed=204FA128FA77A81C \
  -Dtests.class=org.elasticsearch.xpack.watcher.trigger.schedule.tool.CronEvalToolTests \
  -Dtests.method="testEnsureDateIsShownInRootLocale" \
  -Dtests.security.manager=true \
  -Dtests.locale=ar-SS \
  -Dtests.timezone=Asia/Seoul \
  -Dcompiler.java=11 \
  -Druntime.java=11
```

--- a/x-pack/plugin/watcher/src/test/java/org/elasticsearch/xpack/watcher/trigger/schedule/tool/CronEvalToolTests.java
+++ b/x-pack/plugin/watcher/src/test/java/org/elasticsearch/xpack/watcher/trigger/schedule/tool/CronEvalToolTests.java
@@ -56,7 +56,6 @@ public class CronEvalToolTests extends CommandTestCase {
 
     // randomized testing sets arbitrary locales and timezones, and we do not care
     // we always have to output in standard locale and independent from timezone
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/35687")
     public void testEnsureDateIsShownInRootLocale() throws Exception {
         String output = execute("-c","1", "0 0 11 ? * MON-SAT 2040");
         if (ZoneId.systemDefault().equals(ZoneOffset.UTC)) {

